Dockerfile.v4.2.x-armv7l

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:16.04
+LABEL maintainer="Matt Bentley <mbentley@mbentley.net>"
+
+# install omada controller (instructions taken from install.sh); then create a user & group and set the appropriate file system permissions
+RUN \
+  echo "**** Install Dependencies ****" &&\
+  apt-get update &&\
+  DEBIAN_FRONTEND="noninteractive" apt-get install --no-install-recommends -y gosu mongodb net-tools openjdk-8-jre-headless tzdata wget &&\
+  rm -rf /var/lib/apt/lists/* &&\
+  echo "**** Download Omada Controller ****" &&\
+  cd /tmp &&\
+  wget -nv "https://static.tp-link.com/2020/202007/20200714/Omada_SDN_Controller_v4.2.4_linux_x64.tar.gz" &&\
```

Comment:
This should be `https://static.tp-link.com/2020/202011/20201110/Omada_SDN_Controller_v4.2.4_linux_x64.tar.gz`